artichoke-backend/src/hash.rs

```diff
@@ -0,0 +1,14 @@
+use crate::core::BuildHasher;
+use crate::error::Error;
+use std::collections::hash_map::RandomState;
+use crate::ffi::InterpreterExtractError;
+use crate::Artichoke;
+
+impl BuildHasher<RandomState> for Artichoke {
+    type Error = Error;
```

Comment:
To match the change proposed below:

```suggestion
impl BuildHasher for Artichoke {
    type Error = Error;
    type BuildHasher = RandomState;
```